qutebrowser/misc/editor.py

```diff
@@ -134,6 +123,31 @@ def edit(self, text, caret_position=None):
         line, column = self._calc_line_and_column(text, caret_position)
         self._start_editor(line=line, column=column)
 
+    def backup(self):
+        """Create a backup if the content has changed from the original."""
+        if not self._content:
+            return
+        try:
+            fname = self._create_tempfile(self._content)
```

Comment:
Maybe pass a different `prefix` here, i.e. `qutebrowser-editor-backup-` or so? That'd make it easier for users to find the file.